src/pages/Homepage.js

```diff
@@ -19,31 +36,53 @@ const Homepage = () => {
     }
   };
 
+  function updateForecastDegree(event){
+    setDegreeType({
+      degreeType: event.target.value
+    })
+    console.log(setDegreeType)
+  }
+
+  const fahrenheit = Math.round(weatherData.main === undefined ? 0 : weatherData.main.temp );
+  const celsius = Math.round((fahrenheit - 32)* 5/9);
```

Comment:
This would be great for a utility module. Then we can import and reuse all around our application. We can discuss these as well.